fideslog/api/models/models.py

```diff
@@ -53,6 +56,28 @@ class AnalyticsEvent(Base):
     )
 
 
+class Registration(Base):
+    """
+    The persisted details about a registration.
+    """
+
+    __tablename__ = "REGISTRATIONS"
+
+    id = Column(
+        "REGISTRATION_ID", Integer, Sequence("registration_id_seq"), primary_key=True
```

Comment:
Just to confirm, this is what we know in `fides` as the `analytics_id` correct?

Reply:
Nope - `client_id` == `analytics_id` in fides